src/opm/output/eclipse/AggregateWellData.cpp

```diff
@@ -348,7 +348,7 @@ namespace {
                             std::end  (xw.connections),
                     [](const Opm::data::Connection& c)
                 {
-                    return c.rates.any();
+                    return c.rates.any_set();
```

Comment:
Personally I would have preferred to use: `return c.rates.flowing()` - however then the regressions fails for the initial restart file with `IWEL[8] = -1` instead of `IWEL[8] = IWEL[7]` which holds in current master.